### PR TITLE
refactor `listIncompleteUploads` to TypeScript

### PR DESCRIPTION
**Pull request by trim21**
function signature didn't change, no need to update documents and examples

### CODE REVIEW

**Comment by trim21** [comment]
@prakashsvmx

File: src/internal/xml-parser.ts [hunk 1]
```diff
@@ -263,6 +269,72 @@ export function parseTagging(xml: string) {
...
+  }
+  xmlobj = xmlobj.ListMultipartUploadsResult
+  if (xmlobj.IsTruncated) {
+    result.isTruncated = xmlobj.IsTruncated
+  }
+  if (xmlobj.NextKeyMarker) {
+    result.nextKeyMarker = xmlobj.NextKeyMarker
+  }
+  if (xmlobj.NextUploadIdMarker) {
+    result.nextUploadIdMarker = xmlobj.nextUploadIdMarker
```
Comment (prakashsvmx) [inline src/internal/xml-parser.ts:314]:
```suggestion
 result.nextUploadIdMarker = xmlobj.nextUploadIdMarker || ''
 
 
reason is :
```
TypeError: uploadIdMarker should be of type "string"
    at Client.listIncompleteUploadsQuery (/home/prakash/MinIO/GIT/sdks/minio-js/dist/main/internal/client.js:866:13)
```

Comment (prakashsvmx) [reply src/internal/xml-parser.ts:314]: I validated with S3. as it is not applicable in MinIO

File: src/internal/xml-parser.ts [hunk 2]
```diff
@@ -263,6 +269,72 @@ export function parseTagging(xml: string) {
...
+  }
+  xmlobj = xmlobj.ListMultipartUploadsResult
+  if (xmlobj.IsTruncated) {
+    result.isTruncated = xmlobj.IsTruncated
+  }
+  if (xmlobj.NextKeyMarker) {
+    result.nextKeyMarker = xmlobj.NextKeyMarker
+  }
+  if (xmlobj.NextUploadIdMarker) {
+ result.nextUploadIdMarker = xmlobj.nextUploadIdMarker || ''
```
Comment (trim21) [inline src/internal/xml-parser.ts:314]:
```suggestion
    result.nextUploadIdMarker = xmlobj.nextUploadIdMarker || ''
```

**Review by prakashsvmx — APPROVED** [review]
Verified. LGTM :+1:

**Comment by trim21** [comment]
@kaankabalak

**Review by prakashsvmx — APPROVED** [review]

**Review by kaankabalak — APPROVED** [review]

**Pull request merged**